modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -144,10 +143,10 @@ def find_lighthouse_claim!(claim_id:)
         def find_bgs_claim!(claim_id:)
           return if claim_id.blank?
 
-          bgs_service.ebenefits_benefit_claims_status.find_benefit_claim_details_by_benefit_claim_id(
-            benefit_claim_id: claim_id
+          local_bgs_service.find_benefit_claim_details_by_benefit_claim_id(
+            claim_id
           )
-        rescue Savon::SOAPFault => e
+        rescue Faraday::HttpError::NotFound => e
```

Comment:
huh, these never get hit. if you provide an invalid Claim ID, it returns an empty object.

```
(byebug) local_bgs_service.find_benefit_claim_details_by_benefit_claim_id(claim_id)
{}
```